catalog/spatial/stanag4559/spatial-stanag4559-source/src/test/java/org/codice/ddf/spatial/ogc/stanag4559/catalog/source/client/Client.java

```diff
@@ -0,0 +1,429 @@
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.AbsTime;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.AbsTimeHelper;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Coordinate2d;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.DAG;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.DAGHolder;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.DAGListHolder;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Date;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.FileLocation;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NameListHolder;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NameName;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NameValue;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Node;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.NodeType;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Rectangle;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UCO.Time;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UID.Product;
+import org.codice.ddf.spatial.ogc.stanag4559.catalog.common.UID._ProductStub;
+import org.omg.CORBA.IntHolder;
+import org.omg.CORBA.ORB;
+
+public class Client {
+
+    private static Library library;
+
+    private static CatalogMgr catalogMgr;
+
+    private static OrderMgr orderMgr;
+
+    private static ProductMgr productMgr;
+
+    private static DataModelMgr dataModelMgr;
+
+    private static final AccessCriteria accessCriteria = new AccessCriteria("", "", "");
+
+    private static final String SERVER_PATH = "http://localhost:";
+
+    private static final String IOR_PATH = "/data/ior.txt";
+
+    public static void main(String args[]) throws Exception {
+
+        if (args.length != 1) {
+            System.out.println("Unable to obtain IOR File :  No port specified.");
+        }
+
+        org.omg.CORBA.ORB orb = org.omg.CORBA.ORB.init(args, null);
+        int port = Integer.parseInt(args[0]);
+
+        // Get the ior.txt file
+        String iorFile = getIorTextFile(port);
+
+        // Get the Library
+        initLibrary(orb, iorFile);
+
+        // Get the Managers from the Library
+        getManagerTypes();
+        initManagers();
+
+        // CatalogMgr
+        Query query = new Query("NSIL_ALL_VIEW", "");
+        getHitCount(query);
+        DAG[] results = submit_query(query);
+        processAndPrintResults(results);
+
+        // OrderMgr
+        validate_order(orb);
+        PackageElement[] packageElements = order(orb);
+
+        // ProductMgr
+        // For each packageElement in the order response, get the parameters and
+        // related files for the product.
+        for (int i = 0; i < packageElements.length; i++) {
+            Product product = packageElements[i].prod;
```

Comment:
Probably minor, but I tend to use the for each syntax vs this (e.g for PackageElement pkgElement:packageElements)